portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,29 @@
+body{
+	font-family: verdana;
+}
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+
+}
+
+#Project-container {
```

Comment:
General comment for entire css file :
1. Change all ids to classes.
2. Maintain consistency of space in code - always leave only 2 spaces difference between a parent-element and its sub-elements.
3. CSS properties for each element should be written in alphabetical order.